app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+
+    if len(parts) != 3 or parts[0] != "cp" or parts[1] == parts[2]:
+        raise ValueError("Invalid command format")
+
+    try:
+        with open(parts[1], "r") as file, open(parts[2], "w") as copy:
+            copy.write(file.read())
+    except FileNotFoundError:
+        print(f"File: {parts[1]} not found.")
```

Comment:
Code Style: You have used both double quotes and single quotes in your code. It's recommended to stick to one style of quotes. Double quotes are preferable.